frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -337,29 +353,11 @@ const AccountDetails: FC<Props> = ({ account }) => {
             </Row>
           )}
           <style jsx global>{`
-            .account-info-container {
-              border: solid 4px #e6e6e6;
-              border-radius: 4px;
-            }
-
-            .account-info-container > .row:first-of-type .card-cell-text {
-              font-size: 24px;
-            }
-
-            .account-info-container > .row {
-              border-bottom: 2px solid #e6e6e6;
-            }
-
             .account-card-back {
               background-color: #f8f8f8;
             }
-
-            .transaction-link-icon {
-              width: 15px;
-              margin: 0 0 12px 12px;
-            }
           `}</style>
-        </div>
+        </AccountInfoContainer>
```

Comment:
I don't have strong opinions about CSS libraries out there (which is why I went with the default solution in the first place), so I rely on your judgement here. Yet, worth to mention that we don't want to mix up different approaches in the same code-base, so I wonder why this `.account-card-back` was not refactored?

Reply:
I don't want to mix solutions as well, but I had to make a couple of changes to check out that basic features work.
This specific style was not refactored because it is used in another files as well (and I'd have to change them too), so I decided to leave it to migration step.